aws/resource_aws_appsync_graphql_api.go

```diff
@@ -131,6 +131,11 @@ func resourceAwsAppsyncGraphqlApi() *schema.Resource {
 								appsync.FieldLogLevelNone,
 							}, false),
 						},
+						"exclude_verbose_content": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  true,
```

Comment:
The [AppSync API Reference](https://docs.aws.amazon.com/appsync/latest/APIReference/API_LogConfig.html#appsync-Type-LogConfig-excludeVerboseContent) does not seem to note whether this parameter defaults to `true` or `false`, however when running the existing `TestAccAWSAppsyncGraphqlApi_LogConfig` acceptance testing (prior to this change), the `GetGraphApi` response seems to suggest this should be `false`:

```
{
  "graphqlApi" : {
    "name" : "tf-acc-test-421174126728041840",
    "apiId" : "rndvxx2w2fcc7kvfp6kpkcpesq",
    "authenticationType" : "API_KEY",
    "logConfig" : {
      "fieldLogLevel" : "ERROR",
      "cloudWatchLogsRoleArn" : "arn:aws:iam::--OMITTED--:role/tf-acc-test-421174126728041840",
      "excludeVerboseContent" : false
    },
    "userPoolConfig" : null,
    "openIDConnectConfig" : null,
    "arn" : "arn:aws:appsync:us-west-2:--OMITTED--:apis/rndvxx2w2fcc7kvfp6kpkcpesq",
    "uris" : {
      "REALTIME" : "wss://ufdydp2dtjctvl2apsksgf7tvm.appsync-realtime-api.us-west-2.amazonaws.com/graphql",
      "GRAPHQL" : "https://ufdydp2dtjctvl2apsksgf7tvm.appsync-api.us-west-2.amazonaws.com/graphql"
    },
    "tags" : { },
    "additionalAuthenticationProviders" : null,
    "xrayEnabled" : false
  }
}
```

```suggestion
```

Along with the requisite testing and documentation updates below. 👍